packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -81,7 +82,7 @@ type SimpleIdPattern = {|
 
 export type BackupJob = {|
   ...$Exact<Job>,
-  compression?: 'native',
+  compression?: Compress,
```

Comment:
`compression?: 'native' | 'zstd'`

Reply:
the compression can also takes the empty string as value.

Reply:
No, `undefined` means no compression, if `''` is currently accepted, it's a bug.